app/models/display_ad_event.rb

```diff
@@ -2,7 +2,7 @@
 #        :delete for the relationship.  That means no before/after
 #        destroy callbacks will be called on this object.
 class DisplayAdEvent < ApplicationRecord
-  belongs_to :display_ad
+  belongs_to :billboard, class_name: "DisplayAd", foreign_key: "display_ad_id", inverse_of: :billboard_events
   belongs_to :user, optional: true
```

Comment:
Minor: I don't really know, maybe it's unnecessary, but in my experimental branch, I had the idea to add `alias_attribute` for `:billboard_id`. If we have ever written anywhere `display_ad_id = X`, for example, we can rewrite that kind of thing to `billboard_id = X`, rather than leave `display_ad_id` around forever.

Reply:
That makes sense. I think I'll address that in a separate pr.

Reply:
I suppose we can even `rename_column` eventually, with an alias as a first step.